fossunited/fossunited/doctype/event_free_ticket_applications/event_free_ticket_applications.py

```
@@ -0,0 +1,88 @@
+        if coupon_data.used_count >= coupon_data.max_count:
+            frappe.throw("Reached max count of coupon usage.")
+
+        return coupon_data
+
+    def get_ticket_tier(self, coupon_data):
+        """Derive ticket tier name from coupon info."""
+        if coupon_data.tier == "Other":
+            return f"{coupon_data.other_tier} Free Pass"
+        return f"{coupon_data.tier} Free Pass"
+
+    def create_free_ticket(self, coupon_data, ticket_tier):
```

Ruff (0.14.3)

Unused method argument: `coupon_data`

(ARG002)